### DISCUSSION TITLE
how often does ' Share View ' refresh? [May 14]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2111] [Discord Threads]
> 🙋‍♂️ **Xpressd**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### u should see the changes after u refresh  
#  
> 🙋‍♂️ **Xpressd**  
#### oh that is too bad, i wanted to apply this to a meeting scenario where somebody could fill in information as the meeting progressed from a laptop and then have the table being updated live on a weblink where all meeting attendees could easily see it, or a status board left on a TV in a lobby that would update with changes as they were made to the database   
####    
#### i tried embeding the iframe and write some javascript to refresh  but it doesnt have the ability to close the table side panel in that view. it looks bad when it refeshes i wish i could find an alternative to teams excel spreadsheets for live collaboration I really thought nocodb / airtable would be the one but it doesnt have live view updates 😞  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974495016045142036)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
